portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path

```diff
@@ -129,6 +129,11 @@
 	<td>//*[contains(@class, 'component-link tbar-link') and contains(text(),'Clear')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CLOSE_CONTENT_PERFORMANCE_PANEL</td>
+	<td>//button[contains(@type,'button') and contains(@aria-label, 'Close')]</td>
+	<td></td>
+</tr>
```

Comment:
Maybe this path is not needed, you can use Button#CLOSE

Reply:
seen